.github/workflows/ci.yml

```diff
@@ -24,20 +24,114 @@ on:
 
 jobs:
   check-semantic-version:
-    if: github.event.pull_request.draft == false
+    if: github.ref != 'refs/heads/main'
```

Comment:
Why the removal of the draft pull request condition? I could see wanting both of these conditions maybe

Reply:
Before a PR comes out of draft you should know that it's at least meeting the basic light-to-run tests was the rationale. Honestly I don't like the draft gating anyway; I don't want developers marking things as `ready for review` unless they're fully green, so am tempted to take it off the rest too.